commands/create.go

```diff
@@ -578,7 +608,15 @@ func replaceUserdataFile(userdataContent, customScriptContent []byte, newUserDat
 		}
 
 	default:
-		return fmt.Errorf("existing userdata file does not begin with '#!' or '#cloud-config'")
+		encodedCustomInstallScript, err := gzipEncode(customScriptContent)
+		if err != nil {
+			return err
+		}
+
+		_, err = newUserDataFile.Write([]byte(encodedCustomInstallScript))
+		if err != nil {
+			return err
+		}
```

Comment:
This can be reverted because `encodedCustomInstallScript` should be completely encoded to gzip format and we should be using cloud-config format by default.